framework/encode/vulkan_capture_manager.h

```diff
@@ -1249,10 +1256,7 @@ class VulkanCaptureManager : public ApiCaptureManager
     void PostProcess_vkCmdDebugMarkerInsertEXT(VkCommandBuffer                   commandBuffer,
                                                const VkDebugMarkerMarkerInfoEXT* pMarkerInfo);
 
-    void PostProcess_vkFrameBoundaryANDROID(VkDevice device, VkSemaphore semaphore, VkImage image)
-    {
-        EndFrame();
-    }
+    void PostProcess_vkFrameBoundaryANDROID(VkDevice device, VkSemaphore semaphore, VkImage image) { EndFrame(); }
```

Comment:
What happened to this single-line function and the other two in this file?  Did clang-format collapse them?

Reply:
Yes they were collapsed by clang-format.